network/basics/uri.py

```diff
@@ -278,6 +294,13 @@ def url_filename(url):
         return 'index.html'
     return fn
 
+def load_binary(filename):
```

Comment:
Similarly with naming of this function, it doesn't seem to require a binary file.  So we should probably name it something else.

Also... this seems like a very trivial bit of code to make its own function.  Is there a reason for that?